rasa/core/featurizers/tracker_featurizers.py

```diff
@@ -368,29 +520,68 @@ def load(path: Text) -> Optional["TrackerFeaturizer"]:
         )
         return None
 
+    @staticmethod
+    def _remove_action_unlikely_intent_from_states(states: List[State]) -> List[State]:
```

Comment:
@wochinge Just checking: The why of these functions is pretty clear from the code where they are called, for example - https://github.com/RasaHQ/rasa/blob/8c3e949082fe80fa5718bf21bd8d811d7304deec/rasa/core/featurizers/tracker_featurizers.py#L609

Do you still suggest adding the why in the docstring here?

Reply:
> The why of these functions is pretty clear from the code where they are called, for example -

I agree. 

> Do you still suggest adding the why in the docstring here?
 
I think it's fine. The question is what value does the remaining docstring provide then? I'd rather drop it then having this empty shell of a docstring.

Reply:
I will drop the entire docstring then.

Reply:
Addressed in https://github.com/RasaHQ/rasa/pull/9005. Will resolve this conversation when that PR has been merged into `intent-ted`.